use/profiles.js

```diff
@@ -15,200 +17,191 @@ export const useProfiles = () => {
 
   const genderList = [
     {
-      label: 'Female',
+      label: t('profile.genderList.female'),
       value: 'Female',
     },
     {
-      label: 'Male',
+      label: t('profile.genderList.male'),
       value: 'Male',
     },
     {
-      label: 'Other',
+      label: t('profile.genderList.other'),
       value: 'Other',
     },
   ]
 
   const days = [
     {
-      label: 'Monday',
+      label: t('profile.daysList.monday'),
```

Comment:
Those definitions also can be used anywhere else, not only in profile.

```suggestion
      label: t('Monday'),
```